spec/syntax.md

```diff
@@ -64,7 +64,7 @@ The syntax specification takes into account the following design restrictions:
 
 ## Messages and their Syntax
 
-The purpose of MessageFormat is the allow content to vary at runtime.
+The purpose of MessageFormat is to allow content to vary at runtime.
```

Comment:
Good catch. Should we fix this separately? I can fast track it.

Reply:
Fast-tracking this typo fix separately SGTM.

Reply:
Done